<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnDefaultPanel</class>
 <widget class="QWidget" name="ColumnDefaultPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="exprGroup">
     <property name="title">
      <string>Default value:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="SqlEditor" name="exprEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="namedWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="namedCheck">
        <property name="text">
         <string>Named constraint:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="namedEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqlEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>sqleditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
